I.write-tests/02-remove-vowels/remove-vowels.js

```diff
@@ -1,18 +1,33 @@
 function removeVowels(word) {
-    var characters = word.split('');
+  var characters = word.split("");
+  //const vowelLetters = ["a", "A","o","O","i","I","e","E","u","U","y","Y"];
+  var result = [];
 
-    var result = [];
-
-    characters.forEach(function (character) {
-        if (character === 'a' || character === 'o' || character === 'i' || character === 'e'
-            || character === 'u' || character === 'y') {
-            result.push(character)
-        } else {
-            result.push('_')
-        }
-    })
-
-    return result.join('');
+  //   characters.forEach(function(character) {
+  //    // if (vowelLetters.includes(character))
+  //    if (vowelLetters.includes(character)) {
+  //       result.push("");
+  //     } else {
+  //       result.push(character.toLowerCase());
+  //     }
+  //   });
```

Comment:
These commented out code does not provide and value to the function and therefore can be removed :+1:

Reply:
I have removed the comments 👍